### DISCUSSION TITLE
hide pin projects section

### DISCUSSION THREAD

**Discussion by HossamMuhammedOmar** [community/community#33396] [New to GitHub] [answered]
**Hello** 
How can I hide the pin projects section from my GitHub page?

**Comment by radar-blip** [comment, marked as answer]
You just need to un-pin them. 👍🏻 And they be gone.

**Reply by HossamMuhammedOmar** [reply]
**Thank you** @linuxlawson , but when I do this, GitHub says :

> Your popular repositories will now be shown instead of your pins.

Is there any way to hide it also? I only need my contribution activity. 
Is this possible?

**Reply by radar-blip** [reply]
You have these options here in settings:
If you use this, dont forget to update preferences.
![profiles](https://user-images.githubusercontent.com/39196767/190889356-64b8e938-1529-45c3-8766-f23547b76c64.png)

**Reply by HossamMuhammedOmar** [reply]
Okay, thanks very much

**Reply by radar-blip** [reply]
You're welcome. Hope it works for you.

**Comment by abdulvasay-234** [comment]
you need to click on customize your pin in your pinned section and then you need to uncheck the tick mark to hide or make new pin.

**Reply by HossamMuhammedOmar** [reply]
**Thank you** @afnanabdulvasay-234 , but when I do this, GitHub says :

> Your popular repositories will now be shown instead of your pins.

Is there any way to hide it also? I only need my contribution activity. 
Is this possible?
